lib/synadart.dart

```diff
@@ -2,6 +2,4 @@
 /// educational purposes.
 library synadart;
 
-export 'src/activation.dart';
-
-export 'src/networks/sequential.dart';
+export 'src/export.dart';
```

Comment:
Instead of having a separate `export.dart` file per each directory, could we just export everything relevant in `lib/synadart.dart`?